pipeline/outputs/cloudwatch.md

```diff
@@ -84,7 +84,8 @@ The following AWS IAM permissions are required to use this plugin:
 		"Action": [
 			"logs:CreateLogStream",
 			"logs:CreateLogGroup",
-			"logs:PutLogEvents"
+			"logs:PutLogEvents",
+                        "logs:putRetentionPolicy"
```

Comment:
Is the case important here?

Reply:
Nope, I've just noticed. It can be changed to `PutRetentionPolicy` for consistency